src/__tests__/autocapture.js

```diff
@@ -26,7 +26,7 @@ describe('Autocapture system', () => {
         let div, div2, input, sensitiveInput, hidden, password
         beforeEach(() => {
             div = document.createElement('div')
-            div.className = 'class1 class2 class3'
+            div.className = 'class1 class2 class3          ' // Lots of spaces might mess things up
```

Comment:
Nitpick: Ideally we could write a separate test here, e.g.

```
it('when extra spaces in class names', () => {
  given('className', () => 'class1   class2     ')

  expect(...)
})
```

However this test isn't using given2 yet so let's do the easy thing for now.